src/common/types/feed.js

```diff
@@ -40,6 +40,8 @@ type Comment = {
     message?: string,
     modified_at: string,
     permissions: BoxCommentPermission,
+    replies?: Array<Comment>,
+    repliesTotal?: number,
```

Comment:
Will the new reply be exactly the `Comment` type as it is today or will the type be slightly different? Also, we could consider creating a common type for this to be used in `Comment` and `Annotation`

Reply:
Every reply will be basically a comment from api perspective